crates/astria-conductor/src/conductor.rs

```diff
@@ -319,6 +314,49 @@ impl Conductor {
     }
 }
 
+async fn get_executor(
+    cfg: &Config,
+    block_rx: UnboundedReceiver<crate::executor::ExecutorCommand>,
+    shutdown_rx: Receiver<()>,
+) -> eyre::Result<Executor> {
+    let hook = if cfg.enable_optimism {
```

Comment:
Can simplify this `if else` by moving the optimism hook construction into a free standing function.